requirements.txt

```diff
@@ -67,3 +67,5 @@ webob==1.8.5
 webtest
 zake==0.2.2
 zipp<=1.0.0,>=0.5
+git+https://github.com/StackStorm/st2-rbac-backend.git@master#egg=st2-rbac-backend
+git+https://github.com/StackStorm/st2-auth-ldap.git@master#egg=st2-auth-ldap
```

Comment:
Check the instructions for `requirements.txt` generation described here:
https://github.com/StackStorm/st2/blob/fedeb583093fed35bf98a686c835ebff496228d1/requirements.txt#L1-L7

that should help.

When adding them to `fixed-requirements.txt`, please also make sure to include them in an alphabetic order.